docs/contrib/quality.md

```diff
@@ -0,0 +1,38 @@
+# Quality checks
+
+There are a few checks that we apply to the configuration of this
```

Comment:
I would delete the "that", so the sentence would read like this: "There are a few checks we apply to the configuration..."

Reply:
That is indeed more elegant, but I think that including "that" (or, where appropriate, replacing it with ", which") improves the reading flow for non-native speakers. I would prefer to leave this as-is.